modules/apps/static/portal-configuration/portal-configuration-persistence-test/src/testIntegration/java/com/liferay/portal/configuration/persistence/listener/test/ConfigurationModelListenerTest.java

```diff
@@ -222,13 +222,13 @@ public void onBeforeSave(
 		}
 	}
 
-	private static Configuration _getConfiguration(String pid, String location)
+	private static Configuration _getConfiguration(String pid)
 		throws IOException {
 
 		return OSGiServiceUtil.callService(
 			_bundleContext, ConfigurationAdmin.class,
 			configurationAdmin -> configurationAdmin.getConfiguration(
-				pid, location));
+				pid, StringPool.QUESTION));
```

Comment:
Oh, haha, I was reading the change as the opposite at the time.  It was a long Friday.